package.json

```diff
@@ -43,7 +43,7 @@
     "chokidar-cli": "^3.0.0",
     "cross-env": "^7.0.3",
     "eslint": "^8.36.0",
-    "fs-extra": "^10.0.0",
+    "fs-extra": "^11.1.1",
```

Comment:
Can you revert the dep changes please? While it's useful, I think we should do it in a separate PR :)

Reply:
Fixed. It was due to the fact that I created the MR from main branch instead of release one, then targeted the release branch.